tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -113,6 +117,9 @@ public void run() {
   public final void after() throws Exception {
     mExecutorsForClient.shutdown();
     ConfigurationTestUtils.resetConfiguration();
+    RetryHandlingBlockWorkerClientTestUtils.reset();
```

Comment:
It is in AbstractLocalAlluxioCluster#resetClientPools . I didn't make it a util function because this seems to be the only one that creates a local cluster outside of LocalClusterResource